### PR TITLE
F t30974 continue with color brand

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://yaits.demos-deutschland.de/T30974

- Adjust Colors variables.

### CODE REVIEW

File: demosplan/DemosPlanCoreBundle/Logic/LinkMessageSerializable.php [hunk 1]
```diff
@@ -49,8 +49,6 @@ public function __construct(
      * @param string $routeName
      * @param array  $routeParameters
      * @param string $linkText
-     *
-     * @return LinkMessageSerializable
```
Comment (spiess-demos) [inline demosplan/DemosPlanCoreBundle/Logic/LinkMessageSerializable.php:53]: I guess the integration branch must be merged into this branch f_T30974_continue_with_color_brand to get a clean diff.

Comment (ahmad-demos) [reply demosplan/DemosPlanCoreBundle/Logic/LinkMessageSerializable.php:53]: Done in https://github.com/demos-europe/demosplan-core/pull/876/commits/1ac46ddf6e57dac646ff3d7197cf9ef6fde5334f

**Review by spiess-demos — APPROVED** [review]
lgtm!

**Review by spiess-demos — APPROVED** [review]
lgtm

**Pull request merged**